CHANGES.txt

```diff
@@ -15,6 +15,10 @@ the appropriate section of the docs.
 Breaking Changes
 ================
 
+ - Duplicate definition of settings is not allowed in ``crate.yml`` file
```

Comment:
this entry belongs directly to the breaking changes in the release notes of the version that introduced that new restriction not here.

what belongs into the changelog here is a fix entry which says something like: fixed an issue which prevented crate from starting when path.home was set on the command line

be sure to backport